content/en/contributing/multi-account-region-testing.md

```diff
@@ -0,0 +1,84 @@
+
+## Tips to make your changes compatible with multi-account and multi-region
+
+In order to make your changes compatible with multi-account and multi-region, you should follow the below tips:
+
+- For cross-account, in inter-service-communication for many integrations, you can specify a role, with which permissions the source service makes a request to the target service, to access another service's resource. 
+This role should be in source account. 
+When writing an AWS validated test case, you need to properly configure IAM roles.
+
+    For example: 
+    In the test case `test_api_gateway_kinesis_integration` we have specified a [role](https://github.com/localstack/localstack/blob/ae31f63bb6d8254edc0c85a66e3c36cd0c7dc7b0/tests/aws/services/apigateway/test_apigateway_basic.py#L2017-L2022) in the apigateway which has permissions to access the target kinesis account.
+    ```python
+    result = self.connect_api_gateway_to_kinesis(
+            client,
+            api_name,
+            stream_name,
+            region_name,
+            role_arn,
+        )
+    ```
+
+- While having an inter service communication in cross-account, you can create the client using `with_assumed_role(…)`:
+
+    For example:
+    ```python
+    result = self.connect_api_gateway_to_kinesis(
+            client,
+            api_name,
+            stream_name,
+            region_name,
+            role_arn,
+        )
+    ```
+    
+- When there is no role specified, you should use the source arn conceptually if cross-account is allowed. 
+
+    For example:
+    This can be seen in a case where we [added](https://github.com/localstack/localstack/blob/ae31f63bb6d8254edc0c85a66e3c36cd0c7dc7b0/localstack/utils/aws/message_forwarding.py#L42) `account_id` to [send events to the target](https://github.com/localstack/localstack/blob/ae31f63bb6d8254edc0c85a66e3c36cd0c7dc7b0/localstack/utils/aws/message_forwarding.py#L31) service like sqs, sns, lambda, etc. 
+
+- You should always refer the official AWS API docs and check how the the application is communicating to each other. 
+    
+    For example: 
+    Firehose to s3 refer to [Offical AWS Docs](https://docs.aws.amazon.com/firehose/latest/dev/controlling-access.html#cross-account-delivery-s3).
+
+
+## Test changes in CI with non-default credentials
+
+We regularly run the circleci test jobs of the LocalStack community repository to check the compatibility of cross-account against the changes. 
+To achieve that, we have a [scheduled workflow](https://github.com/localstack/localstack/blob/master/.circleci/config.yml) on [LocalStack](https://github.com/localstack/localstack), which executes the tests with randomised account and region credentials every night at 1:00am UTC.
+
+To manually trigger the scheduled workflow through circleci UI(in case you have the permissions), in order to test your changes with randomised account and region credentials, you can perform the following steps: 
```

Comment:
```suggestion
To manually trigger the workflow through the webinterface of CircleCI (in case you have the permissions), in order to test your changes with randomized account and region credentials, you can perform the following steps: 
```